components/description/ingredients.js

```diff
@@ -0,0 +1,20 @@
+function Ingredients(props){
+  const { ingredients } = props;
+
+  return(
+    <>
+      {ingredients.map((ingredient) => {
```

Comment:
You need a return() after opening the map function brackets {}
Perhaps to test if your component does display data, copy data objects from Mongo Compass, pass them as props to your components